public/lib/color_theme.js

```diff
@@ -0,0 +1,60 @@
+import color from 'color';
```

Comment:
Hi @mmguero , because you adapted the code from Kibana, I think you should put a reference to the original file here.
This will be helpful for future releases and will make debugging easier.

Reply:
Done, see commit 37d791a